File: src/RentCars.Domain/Vehicles/VehiclePhotoBlank.cs
namespace RentCars.Domain.Vehicles
{
    public class VehiclePhotoBlank
    {
        public Guid Id { get; set; }
        public Guid VehicleId { get; set; }
        public String Path { get; set; }
        public Boolean IsDeleted { get; set; }
    }
}

File: API/api.client/src/domain/vehicles/vehiclePhotoBlank.ts
import { VehiclePhoto } from "./vehiclePhoto";

export class VehiclePhotoBlank {
    constructor(
        public id: string,
        public vehicleid: string,
        public path: string,
        public isDeleted: boolean,
    ) { }
    
    public static fromDomain(vehiclePhotos: VehiclePhoto[]) {
        return vehiclePhotos.map(ph => new VehiclePhotoBlank(ph.id, ph.vehicleid, ph.path, false))
    }
}
